std/experimental/allocator/building_blocks/bitmapped_block.d

```diff
@@ -996,22 +996,24 @@ struct BitmappedBlockWithInternalPointers(
     }
 
     /// Ditto
-    void[] resolveInternalPointer(void* p)
+    Ternary resolveInternalPointer(void* p, ref void[] result)
     {
+        result = null;
```

Comment:
On second thought I see you have two more paths that use this, so never mind. Looks good.